libs/sitemaps/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@hicommonwealth/sitemaps",
+  "private": "true",
+  "version": "0.1.0",
+  "description": "Sitemaps generator and related code",
+  "files": [
+    "build"
+  ],
+  "main": "build/index",
+  "types": "src/index",
+  "scripts": {
+    "build": "tsc -b",
+    "clean": "rm -rf build",
+    "check-types": "tsc --noEmit",
+    "test": "echo No tests for @hicommonwealth/sitemaps"
```

Comment:
Seems like you have `test/integration` that should run when we execute tests in libs no?